config/permissions.php

```diff
@@ -142,12 +188,14 @@
     ),
     'manager' => array(
         //user tasks report: enable user field to check any user
-        "/userTasksReport.php", "/services/getTasksFiltered.php",
+        "/userTasksReport.php",
+        "/services/getTasksFiltered.php",
         //project details reports: check any project
-        '/projectDetailsReport.php', '/projectDetails.php',
+        '/projectDetailsReport.php',
+        '/projectDetails.php',
         '/services/getProjectUserCustomerReportJsonService.php',
         //user details report: check any user
         '/viewUserDetails.php',
         '/services/getUserProjectCustomerReportJsonService.php'
     )
-);
+);
```

Comment:
Can we have a commit just for the code styling changes? To avoid mixing it with the real changes.

Reply:
I'm backing out any formatting changes for the moment, as they are distracting and not really necessary